media/js/mozorg/contribute/taskview.js

```diff
@@ -22,20 +22,17 @@
      * https://developer.mozilla.org/en-US/docs/Web/API/Page_Visibility_API
      */
     function getVisibilityStateEventKeyword() {
-        // for browser that support the event unprefixed
-        if (typeof document.hidden !== 'undefined') {
-            return 'visibilitychange';
-        }
+        var evtName = undefined;
 
-        // for Chrome 13 and lower
-        if (typeof document.webkitHidden !== 'undefined') {
-            return 'webkitvisibilitychange';
+        if (typeof document.hidden !== 'undefined') {
+            // for browser that support the event unprefixed
+            evtName = 'visibilitychange';
+        } else if (typeof document.webkitHidden !== 'undefined') {
+            // for Chrome 13 and lower
+            evtName = 'webkitvisibilitychange';
         }
 
-        // for older versions of IE
-        if (typeof document.msHidden !== 'undefined') {
```

Comment:
So, would this not be needed for IE10? Although it is not mentioned here[1], it is mentioned in this article[2] that IE used a ms prefixed property when implementing page visibility in IE10

[1] https://developer.mozilla.org/en-US/docs/Web/API/Page_Visibility_API
[2] https://www.nczonline.net/blog/2011/08/09/introduction-to-the-page-visibility-api/